### ISSUE TITLE
Hi，I met a problem as shown below  when i use try_enqueue_bulk(prod_token, item_first, count)  

### ISSUE THREAD

**Issue by qbq600** [cameron314/concurrentqueue#276]
What kind of input parameters would cause such assertion failed ? I try to reproduce  it but i failed.
![1983195403](https://user-images.githubusercontent.com/94610494/148388129-5f093d50-9f95-4956-86ab-c268dd461974.jpg)

**Comment by cameron314** [comment]
Are you using the same token from multiple threads?

**Comment by qbq600** [comment]
>



> 

yes, I create a producer token in producer thread and a consumer  token in consumer thread.

**Comment by cameron314** [comment]
Sounds like that's still only one thread using each token. Based on the information provided, there's no reason for the queue to assert.

Can you share some code?

**Comment by qbq600** [comment]
> Sounds like that's still only one thread using each token. Based on the information provided, there's no reason for the queue to assert.
> 
> Can you share some code?

ok,I will upload the code tomorrow.
